cfg/windows.cfg

```diff
@@ -5098,6 +5098,70 @@ HFONT CreateFont(
       <not-uninit/>
     </arg>
   </function>
+  <!-- HLOCAL WINAPI LocalAlloc(
+  _In_ UINT   uFlags,
+  _In_ SIZE_T uBytes); -->
+  <function name="LocalAlloc">
+    <noreturn>false</noreturn>
+    <returnValue type="HLOCAL"/>
+    <arg nr="1">
+      <not-uninit/>
+    </arg>
+    <arg nr="2">
+      <not-uninit/>
```

Comment:
It would be a good idea with `<not-bool/>` here. The mistake we primarily detect with `<not-bool/>`, misplaced parentheses, happens for the last argument in a function call:

    if (x(LocalAlloc(a,(b+c)!=0)))

Reply:
Thanks, good to know. I will add it there.
Does it make sense to add it for other arguments where a bool makes no sense too? For example for arguments where the function expects only a pointer or a number, but never a bool.

Reply:
yes I think that does make sense also.